yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why was this committed?

Reply:
Removed!

Reply:
The `website/yarn.lock` is important. Just remove the empty one you created.